go/libkb/login_state.go

```diff
@@ -236,6 +236,43 @@ func (s *LoginState) GetPassphraseStreamForUser(ui SecretUI, username string) (p
 	return nil, err
 }
 
+// GetPassphraseStreamWithPassphrase either returns a cached, verified
+// passphrase stream (maybe from a previous login) or generates a new one via
+// Login. It will return the current Passphrase stream on success or an error
+// on failure.
+func (s *LoginState) GetPassphraseStreamWithPassphrase(passphrase string) (pps *PassphraseStream, err error) {
```

Comment:
It looks like pps return value always ignored?  If so, don't return it?  And then maybe this should be renamed to something like EnsurePassphraseStream.